src/openrct2/paint/Supports.cpp

```diff
@@ -446,6 +446,35 @@ static constexpr const uint16_t word_97B3C4[] = {
 
 // clang-format on
 
+static WoodenSupportSubType rotatedWoodenSupportSubTypes[6][NumOrthogonalDirections] = {
+    { WoodenSupportSubType::NeSw, WoodenSupportSubType::NwSe, WoodenSupportSubType::NeSw, WoodenSupportSubType::NwSe },
```

Comment:
could you add a , on the last item of both of these sub arrays then clang format will make them all the same.